microsoftgraph/microsoft-graph-explorer-v4
# Fix: Narrator announces more information about a sample query header

## Overview
Fixes #1149 

### Demo

Screen reader will announce '_Getting Started has 8 results 1 of 28 Collapsed Level 1_'

### Notes

Follow conversation here https://office.visualstudio.com/OC/_workitems/edit/5300729/

## Testing Instructions

1. Open Graph Explorer in Microsoft Edge Dev Browser.
2. Open Narrator using 'Win+ Ctrl + Enter' key.
3. Navigate to the 'Sample queries' tab item in the left side navigation landmark and select it.
4. Navigate to 'Getting started' Button in the menu-list using tab key.
5. Observe the fix
